pokemongo_bot/cell_workers/camp_fort.py

```diff
@@ -29,6 +31,13 @@ def initialize(self):
         self.config_moving_time = self.config.get("moving_time", 600)
 
     def work(self):
+        # Let's make sure we have balls before we sit at a lure!
+        # See also catch_pokemon.py
+        if sum([inventory.items().get(ball.value).count for ball in
+            [Item.ITEM_POKE_BALL, Item.ITEM_GREAT_BALL, Item.ITEM_ULTRA_BALL]]) <= 0:
+            self.logger.info('Not any pokeballs left, refuse to sit at lure!')
+            return WorkerResult.ERROR
```

Comment:
That part should go after the check on "move_until". There is no point displaying that we would not sit there if the task is deactivated or if we are in the moving phase.
Also, does not seem like an error to me, more a WorkerResult.SUCCESS.